### PR TITLE
normalized changelog format and fixed incorrect versions

### PR DESCRIPTION
**Pull request by sanderr**
### Changes to server changelog
- changelog now conforms to the inmanta-tools format
- current version added
- previous version was listed as 0.3.3 but `setup.py` was at 0.2.0 at the time of those changes.

### Changes to client changelog
- changelog now conforms to the inmanta-tools format
- current version added

### CODE REVIEW

File: server/CHANGELOG.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
-# Change Log
+# v 0.2.1 (?)
```
Comment (andraskvr) [inline server/CHANGELOG.md:1]: 0.2.1 is already released:
https://pypi.org/project/inmantals/

**Review by andraskvr — CHANGES_REQUESTED** [review]

File: server/CHANGELOG.md (hunk 1 above)
Comment (sanderr) [reply server/CHANGELOG.md:1]: I see. Does this mean I should update `setup.py` to contain 0.2.2?

Comment (arnaudsjs) [reply server/CHANGELOG.md]: Our other release pipelines bump the version on master to the next minor release. To be consistent, it should be 0.3.0.

**Review by arnaudsjs — CHANGES_REQUESTED** [review]
See comment

File: server/CHANGELOG.md (hunk 1 above)
Comment (sanderr) [reply server/CHANGELOG.md]: Right, we added new features anyway, my reasoning was a bit off.

**Review by andraskvr — APPROVED** [review]

**Review by arnaudsjs — APPROVED** [review]

**Pull request merged**